lsp/.vscode/launch.json

```diff
@@ -25,6 +25,18 @@
             },
             "preLaunchTask": "npm: compile"
         },
+        {
+            "name": "Debug CloudFormation Server in VSCode",
```

Comment:
is "in VSCode" is redundant in a vscode launch config name? "Debug" also is not normally mentioned

Reply:
We can sculpt the entry names over time. We could plausibly have launch tasks that open up the VS or JetBrains sample clients. I think that is not likely, but is why I was explicit with putting "... in VSCode" in the name.

Reply:
if we have jetbrains etc tasks, then "in Jetbrains" makes sense. but don't need to over-verbose the common case to support that

Reply:
That's reasonable. I've updated the names